Sources/AlgoliaSearchClient/Models/Search/Indexing/AltPartialUpdate.swift

```diff
@@ -0,0 +1,256 @@
+//
+//  AltPartialUpdate.swift
+//  
+//
+//  Created by Vladislav Fitc on 31/07/2020.
+//
+
+import Foundation
+
+public enum AltPartialUpdate {
```

Comment:
Should we document this public enum?

Reply:
This is the file I forgot to remove. Thanks 😎